pkg/orm/models_info_f.go

```diff
@@ -241,7 +242,27 @@ checkType:
 			}
 		}
 
-		fieldType, err = getFieldType(addrField)
+		serializerKey := sf.Tag.Get(serializerStructTagName)
+
+		if serializerKey != "" {
+			tags["type"] = "text"
+			fieldType = TypeTextField
+			err = nil
+
+			if fi.auto || fi.pk {
```

Comment:
I think we don't need to check those. Users should be responsible for making sure that the serializer works for this field.